Integrations/Panorama/Panorama.py

```diff
@@ -653,25 +653,27 @@ def panorama_push_status_command():
     Check jobID of push status
     """
     result = panorama_push_status()
-    if result['response']['result']['job']['type'] != 'CommitAll':
+    job = result.get('response', {}).get('result', {}).get('job', {})
+    if job.get('type', '') != 'CommitAll':
         raise Exception('JobID given is not of a Push.')
 
-    push_status_output = {'JobID': result['response']['result']['job']['id']}
-    if result['response']['result']['job']['status'] == 'FIN':
-        if result['response']['result']['job']['result'] == 'OK':
+    push_status_output = {'JobID': job.get('id')}
+    if job.get('status', '') == 'FIN':
+        if job.get('result', '') == 'OK':
             push_status_output['Status'] = 'Completed'
         else:
-            # result['response']['job']['result'] == 'FAIL'
+            # job['result'] == 'FAIL'
```

Comment:
why is this in comment?

@bakatzir can you give your take here?

Reply:
To simplify for the reviewer, can be removed.